app/main.py

```diff
@@ -14,4 +14,6 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    check_phrase = phrase.lower()
+    check_letter = letter.lower()
+    return check_phrase.count(check_letter)
```

Comment:
```suggestion
    return phrase.lower().count(letter.lower())
```
You can do it as one-liner